### PR TITLE
Mention Phoenix.Socket.assign/3 in the channel guide

### PR DESCRIPTION
**Pull request by leikind**
I think it is important to mention socket assigns when talking about channels.

### CODE REVIEW

File: G_channels.md [hunk 1]
```diff
@@ -231,6 +231,16 @@ end
 
 That's all there is to our basic chat app. Fire up multiple browser tabs and you should see your messages being pushed and broadcasted to all windows!
 
+#### Socket Assigns
+
+Similar to connection structs (`%Plug.Conn{}`) it is possible to assign values to a channel socket. `Phoenix.Socket.assign/3` is conveniently imported into a channel module as `assign/3`:
```
Comment (lancehalvorsen) [inline G_channels.md]: I think instead of the parenthesis, we could have `%Plug.Conn{}` set off by commas.
. . . structs, `%Plug.Conn{}`, it is . . .

**Comment by lancehalvorsen** [comment]
Thank you so much for this, @leikind!

I left one tiny comment. If you could address that and then squash the commits, we'll be all set.

Thanks again!

**Comment by leikind** [comment]
Done, please have a look now.

**Comment by lancehalvorsen** [comment]
Great. Thank's @leikind! <3

**Pull request merged**